app/controllers/receptacles_controller.rb

```diff
@@ -30,7 +30,14 @@ def index # rubocop:todo Metrics/AbcSize
   def show
     @source_plates = @asset.source_plates
     respond_to do |format|
-      format.html
+      format.html do
+        @aliquots =
+          if @asset.is_a?(AliquotIndexer::Indexable)
```

Comment:
What does this `AliquotIndexer` do? How come that means it cannot be paginated? 

It looks like `app/views/receptacles/_samples.erb` handles the check `<% indexed = @asset.is_a?(AliquotIndexer::Indexable) %>`  within the erb. Not sure if it is worth having the same approach for both, if possible?

Reply:
I should have asked why it might be important to them.